tests/test_geom_opt.py

```diff
@@ -42,11 +42,8 @@ def test_optimize(architecture, struct_path, expected, kwargs):
         calc_kwargs={"model": MODEL_PATH},
     )
 
-    init_energy = single_point.run("energy")["energy"]
-
     struct = optimize(single_point.struct, **kwargs)
 
-    assert struct.get_potential_energy() < init_energy
```

Comment:
Why do we not want this?

Reply:
because under pressure initial energy of the unrelaxed structure can be smaller than the final one...

Reply:
Ah, true, but perhaps then the pressure test should be separated? It seems a shame to remove one of the relatively few physically meaningful tests we have for the other cases.

Reply:
> Ah, true, but perhaps then the pressure test should be separated? It seems a shame to remove one of the relatively few physically meaningful tests we have for the other cases.

I do not think we get any more info by checking the order if the final energy is correct.

Reply:
Not at the moment, no.

I think when I was writing these tests, we were looking for ways to reduce the arbitrary numbers as much as possible. In an ideal world we'd probably just have physical tests based on the parameters, which is where this came in, as a potential starting point for that.

For the tests that we're trying to distinguish, going much further in a sensible way probably isn't possible though, and you're right that on its own it's not adding anything.